iosApp/iosApp/ContentViewController.swift

```diff
@@ -237,17 +247,18 @@ extension ContentViewController : UITextFieldDelegate {
 
         attachImageName = ""
         let userInput = segmentUserInput(message)
-
+        let command = convertToCommand(command: userInput.0,input: userInput.1)
```

Comment:
I have some ideas on how to refactor some of this user input handling, but I think I'll tackle that in a separate PR in the future.